internal/publish/model/exposure_model_test.go

```diff
@@ -799,3 +1038,72 @@ func TestApplyOverrides(t *testing.T) {
 		})
 	}
 }
+
+func TestDaysFromSymptomOnset(t *testing.T) {
+	now := time.Now().UTC()
+
+	cases := []struct {
+		name  string
+		onset int32
+		check int32
+		want  int32
+	}{
+		{
+			name:  "exact_match",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now),
+			want:  0,
+		},
+		{
+			name:  "next_day",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(24 * time.Hour)),
+			want:  1,
+		},
+		{
+			name:  "next_day_round_down",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(35 * time.Hour)),
+			want:  1,
+		},
+		{
+			name:  "next_day_round_up",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(37 * time.Hour)),
+			want:  2,
+		},
+		{
+			name:  "previous_day",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(-24 * time.Hour)),
+			want:  -1,
+		},
+		{
+			name:  "previous_day_round_down",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(-25 * time.Hour)),
+			want:  -1,
+		},
+		{
+			name:  "previous_day_round_up",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(-47 * time.Hour)),
+			want:  -2,
+		},
+		{
+			name:  "multiple_days",
+			onset: IntervalNumber(now),
+			check: IntervalNumber(now.Add(8*24*time.Hour + 2*time.Hour)),
+			want:  8,
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			got := DaysFromSymptomOnset(tc.onset, tc.check)
+			if tc.want != got {
+				t.Fatalf("wrong day instance between %v and %v, want: %v got: %v", tc.onset, tc.check, tc.want, got)
```

Comment:
got before want

Reply:
yeah... fixed this, but we have this problem everywhere.